samples/snippets/subscriber_test.py

```diff
@@ -23,13 +24,14 @@
 import subscriber
 
 UUID = uuid.uuid4().hex
+PY_VERSION = f"{sys.version_info.major}.{sys.version_info.minor}"
 PROJECT_ID = os.environ["GOOGLE_CLOUD_PROJECT"]
-TOPIC = "subscription-test-topic-" + UUID
-DEAD_LETTER_TOPIC = "subscription-test-dead-letter-topic-" + UUID
-SUBSCRIPTION_ADMIN = "subscription-test-subscription-admin-" + UUID
-SUBSCRIPTION_ASYNC = "subscription-test-subscription-async-" + UUID
-SUBSCRIPTION_SYNC = "subscription-test-subscription-sync-" + UUID
-SUBSCRIPTION_DLQ = "subscription-test-subscription-dlq-" + UUID
+TOPIC = "subscription-test-topic-" + UUID + PY_VERSION
```

Comment:
Probably unnecessary to include PY_VERSION if there's already a UUID.

Reply:
I agree. Let me remove these. I pray for tests still pass.

Reply:
No, they failed randomly for one of the runtimes. I'll leave them in.